### PR TITLE
PEER20-275: 1. Removed Referral Program and Tip from User Profile Settings. 2. At the Authorization data deleted “Telos Active”.

### PR DESCRIPTION
**Pull request by alexeishulga**
1. Removed Referral Program and Tip from User Profile Settings.
2. At the Authorization data deleted “Telos Active”.

### CODE REVIEW

**Comment by ruslanzharkov** [comment]
@alexeishulga please add a descriptive text for pull request title

File: app/translations/en.json [hunk 1]
```diff
@@ -902,6 +902,7 @@
   "app.containers.SignUp.verify": "Verify",
   "app.containers.SignUp.sendAnotherCode": "Send another code",
   "app.containers.SignUp.eosActivePrivateKey": "Telos Active Private Key",
+  "app.containers.SignUp.eosPrivateKey": "Private Key",
```
Comment (ruslanzharkov) [inline app/translations/en.json:905]: `eosPrivateKey` -> `privateKey`

File: app/translations/ru.json [hunk 2]
```diff
@@ -915,6 +915,7 @@
   "app.containers.SignUp.verify": "Подтвердить",
   "app.containers.SignUp.sendAnotherCode": "Отправить другой код",
   "app.containers.SignUp.eosActivePrivateKey": "Telos Active Private Key",
+  "app.containers.SignUp.eosPrivateKey": "Private Key",
```
Comment (ruslanzharkov) [inline app/translations/ru.json:918]: change to russian

Comment (alexeishulga) [reply app/translations/ru.json]: changed: https://github.com/peeranha/peeranha-web/pull/672/commits/8dc9684b33b7ee5f077cca58d021ebe0b5df0630

**Pull request merged**